asynq/decorators.py

```diff
@@ -317,7 +318,24 @@ def decorate(fn):
     return decorate
 
 
-@async_proxy()
+async def asyncio_call(fn, *args, **kwargs):
+    """An asyncio-version of async_call.
+
+    Note that it is not always possible to detect whether an function is a coroutine function or not.
+    For example, a callable F that returns Coroutine is a coroutine function, but inspect.iscoroutinefunction(F) is False.
+
+    """
+    if is_pure_async_fn(fn):
+        return await fn.asyncio(*args, **kwargs)
```

Comment:
Are you sure this is correct?
```
>>> @asynq.asynq(pure=True)
... def f():
...     pass
... 
>>> asynq.is_pure_async_fn(f)
True
>>> f.asyncio

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'PureAsyncDecoratorPythonWrapper' object has no attribute 'asyncio'
```

Reply:
```py
Python 3.11.6 (main, Nov 14 2023, 09:36:21) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> 
>>> import asynq
>>> @asynq.asynq(pure=True)
... def f():
...     pass
... 
>>> asynq.is_pure_async_fn(f)
True
>>> 
>>> f.asyncio
<bound method PureAsyncDecorator.asyncio of @asynq(pure=True) __main__.f>
>>> 
>>> 
```

```py
print(type(f))
<class 'qcore.decorators.decorate.<locals>.PythonWrapper'>
print(f.asyncio)
<bound method PureAsyncDecorator.asyncio of @asynq(pure=True) __main__.f>
```

qcore==1.10.0

Reply:
Maybe try rm -rf __pycache__?

Reply:
Thanks, I may have had an old qcore or something.